maven-central.gradle

```diff
@@ -82,3 +67,8 @@ uploadArchives {
     }
   }
 }
+
```

Comment:
For closing and publishing repo from the command line. Kept the comment to document why the staging Profile ID is not hardcoded here.